frontend/src/components/faq/faq.css

```diff
@@ -45,6 +46,12 @@
   ) !important;
```

Comment:
Using important is not the best practice. Try to avoid it

Reply:
> ** Now the final version look like this **
> ` After adding padding as @Kajol-Kumari said and solving the border issue as @mohit355 said `
> ![Screenshot (189)](https://user-images.githubusercontent.com/59140343/103992017-852c3a80-51b9-11eb-91b6-7ef1e659712b.png)
> 
> ` In case u are wondering why the first and last card dont have curve radius on its top and bottom respectively .Its because of material ui ... anyways it looks good`
> 
> ![Screenshot (190)](https://user-images.githubusercontent.com/59140343/103992021-865d6780-51b9-11eb-971b-995b23b70564.png)

@Kajol-Kumari  is this type of border-radius ( for first and last div) look good to you?  Not looks good to me.

Reply:
I can add radius to them also( first div & last div )   ..Should I add?..